tests/test_subsample.py

```diff
@@ -0,0 +1,97 @@
+"""Tests subsampling
+
+These tests verifies:
+    - Binning correctly
+    - Subsampling correctly
+
+PREREQUISITES
+Spin up Python 3.6 virtualenv, install Python dependencies in requirements.txt
+and Firestore emulator must be running, see PR26 for instructions
+(https://github.com/broadinstitute/scp-ingest-pipeline/pull/26)
+
+Note: When CI environment moves to Python 3.7, tests may break due to minor
+differences in how the reference issues are serialized
+
+# Run all tests in a manner that shows report_issues output
+python3 test_subsample.py -s
+
+"""
+
+import sys
+import unittest
+import numpy as np
+
+sys.path.append("../ingest")
+
+from subsample import SubSample
+
+
+class TestSubsample(unittest.TestCase):
+    AMOUNT_OF_NUMERIC_BINS = 20
+    CLUSTER_PATH = '../tests/data/test_1k_cluster_Data.csv'
```

Comment:
Conventional casing:

```suggestion
    CLUSTER_PATH = '../tests/data/test_1k_cluster_data.csv'
```